SwiftyContextMenu/ContextMenuBlurView.swift

```diff
@@ -0,0 +1,48 @@
+//
```

Comment:
this class shares pretty much all the code with `ContextMenuBackgroundBlurView`. 

Maybe we can create a base class `ContextMenuBlurView` and have two child classes `ContextMenuBackgroundBlurView: ContextMenuBlurView `, `ContextMenuContentBlurView: ContextMenuBlurView` 🤔

Reply:
Yes this makes sense